torch/onnx/_internal/fx/fx_exporter.py

```diff
@@ -63,6 +326,17 @@ def export_fx_to_onnx(
         # with FakeTensorMode.
         with torch.utils._mode_utils.no_dispatch():
             onnxscript_graph = passes.export_fx_to_onnxscript(module, self.options)
+            # ONNX does not support None inputs. During graph building, all None inputs
+            # are removed. Here we register this step to input formatter.
+            self._apply_input_format_step(
+                fx_exporter.RemoveNoneInputStep, fx_module_args, {}
+            )
+            # ONNX can't represent collection types (e.g., dictionary, tuple of tuple of
+            # tensor, etc), we flatten the collection and register each element as output.
+            self._output_formatter.append_step(fx_exporter.FlattenOutputStep())
```

Comment:
I wonder if you imagine steps working kind of like passes, should we create another folder for steps.